README.md

```diff
@@ -31,85 +31,93 @@ The service is subscribed to the following [Keptn CloudEvents](https://github.co
+Create a secret (named `dynatrace` by default) containing the credentials for the Dynatrace Tenant (`DT_API_TOKEN` and `DT_TENANT`) and optionally for the Keptn API (`KEPTN_API_URL`, `KEPTN_API_TOKEN` and `KEPTN_BRIDGE_URL`).
 
-* The `dynatrace-service` can be configured to use a proxy server via the `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables  as described in [`httpproxy.FromEnvironment()`](https://golang.org/pkg/vendor/golang.org/x/net/http/httpproxy/#FromEnvironment). As the `dynatrace-service` connects to a `distributor`, a `NO_PROXY` entry including `127.0.0.1` should be used to prevent these from being proxied. The `HTTP_PROXY` and `HTTPS_PROXY` environment variables can be configured using the `dynatraceService.config.httpProxy` (default `""`) and `dynatraceService.config.httpsProxy` (default `""`) in [values.yml](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/chart/values.yaml), `NO_PROXY` is set to `127.0.0.1` by default. For example:
+```console
+kubectl -n keptn create secret generic dynatrace \
+--from-literal="DT_API_TOKEN=$DT_API_TOKEN" \
+--from-literal="DT_TENANT=$DT_TENANT" \
+--from-literal="KEPTN_API_URL=$KEPTN_API_URL" \
+--from-literal="KEPTN_API_TOKEN=$KEPTN_API_TOKEN" \
+--from-literal="KEPTN_BRIDGE_URL=$KEPTN_BRIDGE_URL" \
+-oyaml --dry-run=client | kubectl replace -f -
+```
 
+ If the Keptn credentials are omitted from this main secret, `KEPTN_API_TOKEN` must be provided by the `keptn-api-token` secret. Furthermore, `dynatraceService.config.keptnApiUrl` and optionally `dynatraceService.config.keptnBridgeUrl` must be set when applying the helm chart (see below).
 
-    ```console
-    helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service.tgz --set dynatraceService.config.httpProxy=http://mylocalproxy:1234 --set dynatraceService.config.httpsProxy=https://mylocalproxy:1234
-    ```
+### 3. Deploy the Service
 
-* To deploy the current version of the *dynatrace-service* in your Kubernetes cluster, use the helm chart located in the `chart` directory.
+To deploy the current version of the *dynatrace-service* in your Kubernetes cluster, use the helm chart located in the `chart` directory.
 Please use the same namespace for the *dynatrace-service* as you are using for Keptn, e.g: keptn.
 
-    ```console
-    helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service-$VERSION.tgz
-    ```
-   
-   **Note**: Replace `$VERSION` with the desired version number (e.g., 0.13.0) you want to install.
-   
-   This installs the `dynatrace-service` in the `keptn` namespace, which you can verify using:
+```console
+helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service-$VERSION.tgz
+```
```

Comment:
I think we should provide an example with `VERSION` set to something that works, e.g., 0.15.0